### ISSUE TITLE
JAWS License

### ISSUE THREAD

**Issue by mrosvik** [digdir/designsystemet#1432]
We should have a license to JAWS so we can test our components with it. 

Cantec er leverandør i norge.

**Comment by mrosvik** [comment]
We decided that this is to expensive. We should use the free 1-hour-testing instead.
